homeassistant/components/todoist/calendar.py

```diff
@@ -82,6 +80,9 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_TOKEN): cv.string,
+        vol.Optional(CONF_UTC_OFFSET_HOURS): vol.All(
```

Comment:
In general, we don't want to introduce new configuration options when we can avoid it. Just use the existing home assistant timezone and offset instead of doing this and it will be consistent with other calendar integrations.

Reply:
Also, this can save from passing this information through so many layers -- you can just apply the UTC offset when parsing the date in the date function.  You can probably use `homeassistant.util.dt.start_of_local_day`

Reply:
Oh nice, that's perfect.  I wasn't aware of that util.

Reply:
I still need to make this change, but need to do a bit more testing to make sure things work as expected.  Thanks so much for the review, greatly appreciated.